### ISSUE TITLE
[Nokia][7215] SAI: ERROR PORT issue in syslog

### ISSUE THREAD

**Issue by Blueve** [Azure/sonic-buildimage#8698]
#### Description

SAI: ERROR PORT log detected in syslog.

#### Steps to reproduce the issue:
1. Install image
2. Reboot and wait system up
3. `cat /var/log/syslog | grep ERR`

#### Describe the results you received:
```
Sep  7 07:21:54.686810 str2-7215-acs-1 INFO syncd#/supervisord: syncd 07:21:54 SAI: ERROR PORT xpSaiPort.c:11009 : Invalid parameter received 8.
Sep  7 07:21:54.687784 str2-7215-acs-1 INFO syncd#/supervisord: syncd 07:21:54 SAI: ERROR PORT xpSaiPort.c:11054 : failed for port ID 1
Sep  7 07:21:54.792146 str2-7215-acs-1 INFO syncd#/supervisord: syncd 07:21:54 SAI: ERROR PORT xpSaiPort.c:11009 : Invalid parameter received 8.
Sep  7 07:21:54.793119 str2-7215-acs-1 INFO syncd#/supervisord: syncd 07:21:54 SAI: ERROR PORT xpSaiPort.c:11054 : failed for port ID 2
Sep  7 07:21:54.863737 str2-7215-acs-1 INFO syncd#/supervisord: syncd 07:21:54 SAI: ERROR PORT xpSaiPort.c:11009 : Invalid parameter received 8.
Sep  7 07:21:54.863737 str2-7215-acs-1 INFO syncd#/supervisord: syncd 07:21:54 SAI: ERROR PORT xpSaiPort.c:11054 : failed for port ID 3

......

Sep  7 07:22:00.385821 str2-7215-acs-1 INFO syncd#/supervisord: syncd 07:22:00 SAI: ERROR PORT xpSaiPort.c:11009 : Invalid parameter received 8.
Sep  7 07:22:00.386248 str2-7215-acs-1 INFO syncd#/supervisord: syncd 07:22:00 SAI: ERROR PORT xpSaiPort.c:11054 : failed for port ID 51
Sep  7 07:22:00.500223 str2-7215-acs-1 INFO syncd#/supervisord: syncd 07:22:00 SAI: ERROR PORT xpSaiPort.c:11009 : Invalid parameter received 8.
Sep  7 07:22:00.500692 str2-7215-acs-1 INFO syncd#/supervisord: syncd 07:22:00 SAI: ERROR PORT xpSaiPort.c:11054 : failed for port ID 52
```

#### Describe the results you expected:
No such logs appear.

#### Output of `show version`:
Use image: Image: https://dev.azure.com/mssonic/build/_build/results?buildId=33599&view=results

**Comment by rajkumar38** [comment]
should be fixed with PR #8762

**Comment by radha-danda** [comment]
@Blueve, issue is fixed in 202012 and master branch as well. 
Link to PR (on master branch): https://github.com/Azure/sonic-buildimage/pull/10403
Link to PR (on 202012 branch): https://github.com/Azure/sonic-buildimage/pull/8762

Kindly check and close the issue
